.github/workflows/slash.yml

```diff
@@ -1,3 +1,16 @@
+# The slash workflow handles slash commands
+#
+# Slash commands are given through comments on pull requests
+# and may be used only by individuals with "write" access to
+# the repository (i.e. maintainers).
+#
+# The workflow looks dispatches another workflow named <command>-command
```

Comment:
typo? maybe was supposed to say "...looks for and dispatches to another..."?

Reply:
Good catch, thank you, fixed now